src/ansys/fluent/core/file_session.py

```diff
@@ -258,7 +275,7 @@ def get_fields(self):
 
         for transaction in self._vector_field_transactions:
             if "velocity" not in transaction.field_name:
-                raise RuntimeError("Only 'velocity' is allowed field.")
+                raise InvalidFieldName("Only 'velocity' is allowed field.")
```

Comment:
Again, the caller is providing all the details. Note that the English is not well structured. We would say _the only allowed field is..._ .

Reply:
Yes, English grammar is incorrect at many places including docstring and error messages.

Reply:
Done. Thank you.